### DISCUSSION TITLE
non-editable on a view [Oct 26]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4268] [Discord Threads]
> 🙋‍♂️ **gettohhole**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### currently you can lock the table so that no one can edit it but able to see the value  
#  
> 🙋‍♂️ **gettohhole**  
#### hm 😦 I need to protect some values, that need to be seen, so that users can put data others.  
#  
> 🙋‍♂️ **gettohhole**  
#### Sound's like that's not possible right now, correct?  
#  
> 🙋‍♂️ **gettohhole**  
#### 😦  
#  
> 🙋‍♂️ **gettohhole**  
#### unfortunate.  
#  
> 🙋‍♂️ **wkw**  
#### u mean lock some cells only?  
#  
> 🙋‍♂️ **gettohhole**  
#### yup  
#  
> 🙋‍♂️ **wkw**  
#### seems not possible currently  
#  
> 🙋‍♂️ **wkw**  
#### u may request it in github  
#  
> 🙋‍♂️ **gettohhole**  
#### I have like 10 columns 8 of them have data prefilled and the other two (very related) should be filled by a user.  
#  
> 🙋‍♂️ **gettohhole**  
#### I'll do so. Thanks  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1032192080241496106)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
